tests/integration/cri-containerd/gha-run.sh

```diff
@@ -48,7 +48,7 @@ function install_dependencies() {
 
 	for github_dep in "${github_deps[@]}"; do
 		IFS=":" read -r -a dep <<< "${github_dep}"
-		install_${dep[0]} "${dep[1]}"
+		install_"${dep[0]}" "${dep[1]}"
```

Comment:
Here you could enclose the entire word using double quotes:
```suggestion
		"install_${dep[0]}" "${dep[1]}"
```